marketplace_builder/views/pages/how-to/modules/creating-module.liquid

```diff
@@ -0,0 +1,192 @@
+                    <thead>
+                        <tr>
+                            <th>
+                                Name
+                            </th>
+                            <th>
+                                Properties
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for customization in c.customizations.results %}
+                            <tr>
+                                <td>
+                                    {{ customization.human_name }}
+                                </td>
+                                <td>
+                                    {{ customization.properties }}
+                                </td>
+                            </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </section>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-sm-8">
+            <section class="pages">
+                <h3> Manage Pages </h3>
+                {%- query_graph "get_pages", result_name: "c", id: id -%}
+                <table class="table">
+                    <thead>
+                        <tr>
+                            <th>
+                                Slug
+                            </th>
+                            <th>
+                                Format
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for page in c.pages.results %}
+                            <tr>
+                                <td>
+                                    {{ page.slug }}
+                                </td>
+                                <td>
+                                    {{ page.format }}
+                                </td>
+                            </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </section>
+        </div>
+    </div>
+</div>
+{% endraw %}
+```
+
+As you can see you are using the queries like usual `{%- query_graph "get_pages", result_name: "c", id: id -%}`. The only difference between this code and code of a regular Instance is that this module code is split into public and private folders, and if you'd pull the code when it's installed on an Instance, you would get a structure like this: 
+
+```
+.
+├── module_marketplace
+│   └── views
+|   └── graph_queries
+│       ...       
+└── modules
+    └── admin
+        └── public
+            └── views
+                └── pages
+                    └── admin.html.liquid
+```
+
+
+
+## Live example and source code
+
+You can download the directory structure and all code included in this tutorial [here](https://github.com/mdyd-dev/nearme-documentation/tree/master/marketplace_builder/assets/files/tutorials/modules/admincms.zip) as a zip file.  
```

Comment:
Can we put it into example marketplace and link to source code + working page like we did with couple other guides?

Also i would move zip file there or remove it completely - im pretty sure that in the next 3 code changes zip will become outdated ;)

https://github.com/mdyd-dev/marketplace-nearme-example - this is the repo where we can show off stuff - i can go through tutorial and do it - additional user test wont hurt.

Reply:
@pavelloz That's a great idea! Thank you for offering to go through the tutorial and do it.

Reply:
Ok, let me do this then.
I should make it this week so the next week update/accnouncement could include it